src/services/api/github/useCreatePR.ts

```diff
@@ -0,0 +1,22 @@
+import { TranscriptSubmitOptions } from "@/components/menus/SubmitTranscriptMenu";
+import { SaveToGHData } from "../../../../types";
+import axios from "axios";
+import { useMutation } from "@tanstack/react-query";
+
+type CreatePRProps = Record<string, any> &
+  Partial<
+    Omit<SaveToGHData, "reviewId"> & {
+      prUrl: string;
+      prRepo: TranscriptSubmitOptions;
+      body: string;
+      oldDirectoryList: any;
+    }
+  >;
+
+const createPR = (props: CreatePRProps) => axios.post("/api/github/pr", props);
+
+export function useCreatePr() {
```

Comment:
you use capitalize "PR" on the rest of the code
```suggestion
export function useCreatePR() {
```